spec/requests/users_spec.rb

```diff
@@ -2,34 +2,28 @@
 
 describe 'Users', type: :request do
   describe 'GET #index' do
-    before(:example) { get users_url }
-
     it 'successfull request' do
-      expect(response.status).to eq(200)
+      get '/'
+      expect(response).to be_successful
     end
 
     it 'should render the index template' do
+      get '/users'
       expect(response).to render_template(:index)
     end
-
-    it 'should render the index placeholder' do
-      expect(response.body).to include('List of all users')
-    end
   end
 
-  describe 'GET #show' do
-    before(:example) { get user_url(1) }
+  describe 'GET users from /users/:id to show' do
+    user = User.create!(name: 'Tom', photo: 'https://picsum.photos/200/300', bio: 'I am John', posts_counter: 4)
```

Comment:
Good work implementing some tests! 

- [ ] Kindly fix the error in the terminal


![image](https://github.com/Roland-Ntwali/Blog-App/assets/93630700/53e62fb1-1044-440e-940a-bbb14b9107c2)